mods/ctf_pvp_engine/ctf_chat/init.lua

```diff
@@ -148,6 +158,68 @@ minetest.register_chatcommand("team", {
 	end
 })
 
+minetest.register_chatcommand("access", {
+	description = "Open the team console, or run team command (see /team help)",
+	func = function(name, param)
+		local a_flag, a_team_or_player = string.match(param, "^add ([%a%d_-]+) ([%a%d_]+)")
+		local r_flag, r_team_or_player = string.match(param, "^remove ([%a%d_-]+) ([%a%d_]+)")
+		local c_flag, c_team_or_player = string.match(param, "^clear ([%a%d_-]+) ([%a%d_]+)")
+		local l_flag = string.match(param, "^list ([%a%d_-]+)")
+		local o_flag = string.match(param, "^open ([%a%d_-]+)")
+		local cl_flag = string.match(param, "^close ([%a%d_-]+)")
+		if ctf.player(name).auth or minetest.get_player_privs(name).ctf_admin then
+			if a_flag and a_team_or_player then
+				if ctf.access_add(ctf.player(name).team,a_flag,a_team_or_player) then
+					return true, "Successfully added " .. a_team_or_player .. " to " .. a_flag .. "'s protection"
+				else
+					return false, "Failed to add to access!"
+				end
+			elseif r_flag and r_team_or_player then
+				if ctf.access_remove(ctf.player(name).team,r_flag,r_team_or_player) then
+					return true, "Successfully removed " .. r_team_or_player .. " from " .. r_flag .. "'s protection"
+				else
+					return false, "Failed to remove from access!"
+				end
+			elseif c_flag and c_team_or_player then
+				if ctf.access_clear(ctf.player(name).team,c_flag,c_team_or_player) then
+					return true, "Successfully cleared " .. a_flag
+				else
+					return false, "Failed to add to clear access!"
+				end
+			elseif l_flag then
+				if ctf.access_list(name,ctf.player(name).team,l_flag) then
+					return
+				else
+					return false, "Failed to list access!"
+				end
+			elseif o_flag then
+				if ctf.access_open(ctf.player(name).team,o_flag) then
+					return true, o_flag .. " is open now."
+				else
+					return false, "Failed to open access!"
+				end
+			elseif cl_flag then
+				if ctf.access_close(ctf.player(name).team,cl_flag) then
+					return true, cl_flag .. " is set to white list only."
+				else
+					return false, "Failed to close access!"
+				end
+			elseif param=="help" then
+				access_console_help(name)
+			else
+				if param ~= "" and param ~= nil then
+					minetest.chat_send_player(name, "'"..param.."' is an invalid parameter to /access")
+					access_console_help(name)
+				end
+			end
+		else
+			return false, "You are not the team owner!"
+		end
```

Comment:
To prevent overly long lines, change to 
```lua
if not ctf.player(name).auth and not minetest.check_player_privs(name, "ctf_admin") then
	return false, "You are not the team owner!"
end

if a_flag and a_team_or_player then
	if ctf.access_add(ctf.player(name).team,a_flag,a_team_or_player) then
		return true, "Successfully added " .. a_team_or_player .. " to " .. a_flag .. "'s protection"
	else
		return false, "Failed to add to access!"
	end
elseif r_flag and r_team_or_player then
	if ctf.access_remove(ctf.player(name).team,r_flag,r_team_or_player) then
		return true, "Successfully removed " .. r_team_or_player .. " from " .. r_flag .. "'s protection"
	else
		return false, "Failed to remove from access!"
	end
elseif c_flag and c_team_or_player then
	if ctf.access_clear(ctf.player(name).team,c_flag,c_team_or_player) then
		return true, "Successfully cleared " .. a_flag
	else
		return false, "Failed to add to clear access!"
	end
elseif l_flag then
	if ctf.access_list(name,ctf.player(name).team,l_flag) then
		return
	else
		return false, "Failed to list access!"
	end
elseif o_flag then
	if ctf.access_open(ctf.player(name).team,o_flag) then
		return true, o_flag .. " is open now."
	else
		return false, "Failed to open access!"
	end
elseif cl_flag then
	if ctf.access_close(ctf.player(name).team,cl_flag) then
		return true, cl_flag .. " is set to white list only."
	else
		return false, "Failed to close access!"
	end
elseif param=="help" then
	access_console_help(name)
else
	if param ~= "" and param ~= nil then
		minetest.chat_send_player(name, "'"..param.."' is an invalid parameter to /access")
		access_console_help(name)
	end
end
```